app/elm/Combat.elm

```diff
@@ -11,11 +11,7 @@ import List.Extra as ListX
 
 type alias Model =
     { turnHistory : List Attack
-    , players : List String
-    , characters :
-        { allies : List Creature
-        , enemies : List Creature
-        }
+    , party : List String
```

Comment:
This will eventually have to move out like userId, since all parties will share the same combat state, but that should be straightforward.